### PR TITLE
[Hub Generated] Review request for Microsoft.Cdn to add UrlRewrite and CacheKeyQueryString action to 2019-04-15

### PR DESCRIPTION
**Pull request by hytao**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/hytao/azure-rest-api-specs/dev-cdn-Microsoft.Cdn-2019-06-15-preview...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdn.json [hunk 1]
```diff
@@ -3377,6 +3425,58 @@
...
+            "Exclude",
+            "ExcludeAll"
+          ],
+          "x-ms-enum": {
+            "name": "queryStringBehavior",
+            "modelAsString": true
+          }
+        },
+        "queryParameters": {
+          "description": "query parameters to include or exlude. (comma separated)",
```
Comment (nschonni) [inline specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdn.json]:
```suggestion
          "description": "query parameters to include or exclude. (comma separated)",
```

File: specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdn.json [hunk 2]
```diff
@@ -3241,6 +3243,52 @@
...
+      ],
+      "properties": {
+        "@odata.type": {
+          "type": "string",
+          "enum": [
+            "#Microsoft.Azure.Cdn.Models.DeliveryRuleUrlRewriteActionParameters"
+          ]
+        },
+        "sourcePattern": {
+          "description": "define a request URI pattern that identifies the type of requests that may be rewritten. If value is blank, all strings are matched",
```
Comment (kpajdzik) [inline specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdn.json]:
```suggestion
          "description": "define a request URI pattern that identifies the type of requests that may be rewritten. If value is blank, all strings are matched.",
```

File: specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdn.json [hunk 3]
```diff
@@ -3241,6 +3243,52 @@
...
+          "type": "string"
+        },
+        "destination": {
+          "description": "Define the relative URL to which the above requests will be rewritten by",
```
Comment (kpajdzik) [inline specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdn.json]:
```suggestion
          "description": "Define the relative URL to which the above requests will be rewritten by.",
```

To be consistent :)

File: specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdn.json [hunk 4]
```diff
@@ -3241,6 +3243,52 @@
...
+          "type": "string"
+        },
+        "preserveUnmatchedPath": {
+          "description": "Whether to preserve unmatched path. Default value is true",
```
Comment (kpajdzik) [inline specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdn.json]:
```suggestion
          "description": "Whether to preserve unmatched path. Default value is true.",
```

File: specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdn.json (hunk 1 above)
Comment (kpajdzik) [inline specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdn.json]:
```suggestion
          "description": "query parameters to include or exclude (comma separated).",
```

**Review by kpajdzik — APPROVED** [review]

**Comment by hytao** [comment]
After discussing with PM, we are going to make changes to the stable 2019-04-15 version instead of the 2019-06-15 preview version. I have move all my changes to 2019-04-15 version.

**Pull request merged**